example/test_problem/Hydro/CCSN/Input__TestProb.CoreCollapse

```diff
@@ -20,15 +20,22 @@ CCSN_Eint_Mode   1                                 # Mode of obtaining internal
-CCSN_CC_MaxRefine_LV1     6                        # reduced maximum refinement level 1 to this value [MAX_LEVEL-2]
-CCSN_CC_MaxRefine_LV2     7                        # reduced maximum refinement level 2 to this value [MAX_LEVEL-1]
-CCSN_CC_MaxRefine_Dens1   1.0e11                   # central density threshold that reduces the maximum refinement level 1 [1.0e11]
-CCSN_CC_MaxRefine_Dens2   1.0e12                   # central density threshold that reduces the maximum refinement level 2 [1.0e12]
+CCSN_CC_MaxRefine_Flag1   1                        # enable limiting maximum refinement level 1  [0]
+CCSN_CC_MaxRefine_Flag2   1                        # enable limiting maximum refinement level 2  [0]
+CCSN_CC_MaxRefine_LV1     6                        # reduced maximum refinement level 1 to this value  [MAX_LEVEL-2]
+CCSN_CC_MaxRefine_LV2     7                        # reduced maximum refinement level 2 to this value  [MAX_LEVEL-1]
+CCSN_CC_MaxRefine_Dens1   1.0e11                   # central density threshold that reduces the maximum refinement level 1  [1.0e11]
+CCSN_CC_MaxRefine_Dens2   1.0e12                   # central density threshold that reduces the maximum refinement level 2  [1.0e12]
 CCSN_MaxRefine_RadFac     0.15                     # factor that determines the maximum refinement level based on distance from the box center  [0.15]
 CCSN_CC_CentralDensFac    1.0e13                   # factor that reduces the dt constrained by the central density (in cgs) during the core collapse  [1.0e13]
-CCSN_CC_Red_DT            1.0e-5                   # reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce [1e-5]
+CCSN_CC_Red_DT            1.0e-5                   # reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce  [1e-5]
 CCSN_LB_TimeFac           0.10                     # factor that scales the dt constrained by lightbulb scheme  [0.1]
 
-CCSN_Is_PostBounce        0                        # boolean that indicates whether core bounce has occurred   [0]
+CCSN_CC_Rot               2                        # mode for rotational profile  (0:off, 1:formulated-rotation, 2:IC)                             [2]
+CCSN_CC_Rot_R0            1.03376e8                # characteristic radius, R_0, in the formulated rotational profile
+                                                   # ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )                                 [2.0e8]
+CCSN_CC_Rot_Omega0        2.0                      # central rotational velocity, Omega_0, in the formulated rotational profile
+                                                   # ( Omega(r)=Omega_0*[R_0^2/(r^2+R_0^2)], r: spherical radius )                                 [0.5]
+CCSN_CC_Rot_Fac          -1.0                      # multiplication factor for the rotational profile (only for a tabular rotation)  (<=0 -> off)  [-1.0]
```

Comment:
`# multiplication factor for the tabular rotational profile (<=0.0 -> off)  [-1.0]`

Reply:
I updated the comment.